Wikipedia/Code/ArticleViewController+ArticleStateRestoration.swift

```diff
@@ -0,0 +1,67 @@
+//~~~**DELETE THIS HEADER**~~~
```

Comment:
Need this header deleted. At the time this default was made, Xcode didn't support the ability to have an empty header when a new file was created. It's possible that has since changed.

Right now it adds `DELETE THIS HEADER`, and there's a git commit hook that checks if this exists in any file before  allowing the commit.